pymatgen/analysis/graphs.py

```diff
@@ -1729,6 +1770,14 @@ def with_local_env_strategy(cls, molecule, strategy) -> Self:
         mg.set_node_attributes()
         return mg
 
+    @deprecated(
+        from_local_env_strategy,
+        "Deprecated on 2024-03-29, to be removed on 2025-03-20.",
+    )
+    @classmethod
+    def with_local_env_strategy(cls, *args, **kwargs):
```

Comment:
maybe try swapping the decorator order so that `classmethod` is at the top

Reply:
Thanks. Everything looking good now.